src/Core/Backup/Modifier/HumanReadableBackupAgeModifier.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Backup\Modifier;
+
+use PrestaShop\PrestaShop\Core\Grid\Modifier\ModifierInterface;
+use PrestaShop\PrestaShop\Core\Util\DateTime\TimeDefinition;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class HumanReadableBackupAgeModifier modifies age to be human readable
+ */
+final class HumanReadableBackupAgeModifier implements ModifierInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function modify(array $record)
+    {
+        $record['age'] = $this->getHumanReadableAge($record['age']);
```

Comment:
maybe instead of replacing original value with modified, we better return what's new, like:

```php
return [
    'human_readable_age' => $this->getHumanReadableAge($record['age']),
];
```

in that case, `ModifiersApplicator` would be responsible for merging this change (or many changes if this $record has more than one modifier) into `$record` what way we are S**O**LID?